milli/src/lib.rs

```diff
@@ -1,5 +1,3 @@
-#![allow(clippy::reversed_empty_ranges)]
-#![allow(clippy::too_many_arguments)]
```

Comment:
imo, when we change the code that has clippy error, we should realize that is allowed.

Reply:
@ManyTheFish will be the best to answer 😇